macOS/DuckDuckGo/VisualRefresh/VisualStyleConfigurable.swift

```diff
@@ -41,18 +111,30 @@ final class VisualStyleManager: VisualStyleConfigurable {
 
     private var cancellables: Set<AnyCancellable> = []
 
-    private var isEnabled: Bool {
-        featureFlagger.isFeatureOn(.visualRefresh)
-    }
-
     init(featureFlagger: FeatureFlagger) {
         self.featureFlagger = featureFlagger
 
         subscribeToLocalOverride()
     }
 
-    var toolbarHeight: CGFloat {
-        currentStyle.toolbarHeight
+    // MARK: - VisualStyleConfigurable implementation
+
+    func addressBarHeight(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarHeight(for: type)
+    }
+
+    func addressBarTopPaddig(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarTopPaddig(for: type)
+    }
+
+    func addressBarBottomPaddig(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarBottomPaddig(for: type)
+    }
```

Comment:
I worry that at the end of the project, the `VisualStyleConfigurable` protocol will grow significantly, and there is some code “duplication” (so to speak) here, because you define the same methods in `VisualStyleManager` and in `VisualStyle`. Have you considered conforming `VisualStyle` to `VisualStyleConfigurable` instead, and returning a `VisualStyle` object (or actually `any VisualStyleConfigurable`) from the manager?

Then the call site would look like this:
```swift
manager.currentStyle.addressBarBottomPadding(for: .default)
```

If you think that `currentStyle` in the call site is superfluous and you prefer the way you’re doing it now, that’s completely fine by me – but I just wanted to point out that `VisualStyle` should also conform to `VisualStyleConfigurable`.

Reply:
I like this idea. I switched, and the manager will be mainly in charge of passing the currently selected style.